app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[1] == command_list[2]:
+        return
+    else:
+        with open(command_list[1], "r") as file_in,\
```

Comment:
It's better to use `()` than `\` to break lilnes.

Reply:
I can't use () in a right way in this case. It will be in another way)

Reply:
You sure can! All you have to do is to enclose everything after `with` in `()`:
```python
with (open...,
      open...)